test/test_va_api_config_attribs.cpp

```diff
@@ -126,7 +127,9 @@ TEST_P(VAAPIConfigAttribs, GetConfigAttribs)
     // can get the supported values for them.
     ConfigAttributes supported = actual;
     std::for_each(supported.begin(), supported.end(),
-        [](VAConfigAttrib& s) { s.value = 0; });
+    [](VAConfigAttrib & s) {
+        s.value = 0;
+    });
```

Comment:
Seems astyle struggles with lambda expressions.  Depending on how it's originally formatted, astyle results can differ.  I suppose this one is acceptable, though.